main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
```

Comment:
Consider making this a function, maybe with helper functions to gather user input and print out the story.